.github/workflows/changelog-label-check.py

```diff
@@ -0,0 +1,15 @@
+from github import Github
+import os
+import re
+
+
+repo = Github(os.environ.get('GITHUB_TOKEN')).get_repo(os.environ.get('GITHUB_REPOSITORY'))
+pr_number = int(re.search('refs/pull/([0-9]+)/merge', os.environ.get('GITHUB_REF')).group(1))
```

Comment:
Maybe you can just `os.environ.get('GITHUB_REF').split('/')[2]`

Wondering if there is a way to get the PR number without having to parse GITHUB_REF ?

Edit: Maybe this ? https://github.com/actions/checkout/issues/58#issuecomment-545769838

```
pull_number=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
```